packages/common/src/api/comments.ts

```diff
@@ -87,15 +87,15 @@ const commentsApi = createApi({
       options: { type: 'mutation' },
       async onQuerySuccess(
         { data: newId },
-        { entityId, body, userId, timestampS, parentCommentId },
+        { entityId, body, userId, trackTimestampMs, parentCommentId },
         { dispatch }
       ) {
         const newComment: Comment = {
           id: newId,
           userId,
           message: body,
           isPinned: false,
-          timestampS,
+          trackTimestampMs,
```

Comment:
maybe we should be using seconds? for longer things like podcasts this number can get pretty large. either way should be fine though because max postgres int will store 596 hours in ms. it's just a really big number

Reply:
yeah I think seconds might be better; I'll pivot it in this PR real quick